_posts/2017-03-20-developers-toolbox-q1-2017.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Four Useful Additions to Any Developers Toolbox in Q1 2017
+date: 2017-03-20
+author: Ro Wang
+tags: tips efficiency docker
+featured_image: "/blog/content-authoring-d8.jpg"
+featured_image_alt: "Q1 2017 Tips"
+featured_image_height: "917px"
+featured_image_width: "1474px"
+summary: What we've added to our developer toolboxes in Q1 2017!
+drupal_planet_summary: |
+  The Web Development world evolves quickly, and keeping abreast of new tools and methodologies while simultaneously delivering client work is tough.
+
+  Here's what we've added to our developer toolboxes in Q1 2017.
+
+---
+
+_This post is part of a new series called "Quarterly Tips and Tricks."_
+
+The Web Development world evolves quickly, and keeping abreast of new tools and methodologies while simultaneously delivering client work is tough.
+
+One of our core [philosophies](http://savaslabs.com/mission-and-values/#excel) at Savas Labs is to "_hold ourselves to a high and constantly improving standard_." To that end, we dedicate time each month to share new tips and tools with each other to continue increasing our collective knowledge base -- we call these our :zap: "Lightning Skillshares!" :zap:
+
+Since we believe in the open source ethos and sharing with the Dev community, in this series, we'll highlight some tips we use to improve our day-to-day efficiency (and hopefully yours too!).
+
+What we've added to our toolboxes in Q1 2017:
+
+[1. Using Teleconsole for collaboration](#Using Teleconsole for collaboration)
```

Comment:
@lrowang I found the cause of the test failures, although the error message wasn't exactly helpful! I figured it was something in the HTML of this post since nokogiri, which we use to test our HTML, was throwing the error, so I checked out your post and found these broken heading links. If you inspect the headings you'll see that the IDs are lowercase and use dashes instead of spaces:

<img width="1279" alt="screen shot 2017-03-10 at 11 30 09 am" src="https://cloud.githubusercontent.com/assets/8465998/23803826/c278d9ae-0585-11e7-99c6-14a042326eb0.png">

So you'll just need to update these link paths to `(#using-teleconsole-for-collaboration)`, etc.